### PR TITLE
Usage of zend-expressive-authentication 1.0

### PR DESCRIPTION
**Pull request by ezimuel**
This PR adds the usage of `zend-expressive-authentication` ^1.0. This is required for preparing the release of `zend-expressive-authentication-zendauthentication` 1.0.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,32 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+## 1.0.0 - TBD
+
+### Added
+
+- [#9](https://github.com/zendframework/zend-expressive-authentication-zendauthentication/pull/9)
+  introduces the usage of zend-expressive-authentication ^ 1.0.
```
Comment (michalbundyra) [inline CHANGELOG.md]: Remove space after `^` before the version.

Comment (ezimuel) [reply CHANGELOG.md]: Done, thanks!

**Review by weierophinney — APPROVED** [review]
:ship:

**Comment by weierophinney** [comment]
Thanks, @ezimuel!

**Pull request closed without merging**